cmd/prysmctl/validator/cmd.go

```diff
@@ -0,0 +1,127 @@
+package validator
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/logrusorgru/aurora"
+	"github.com/prysmaticlabs/prysm/v3/cmd"
+	"github.com/prysmaticlabs/prysm/v3/cmd/validator/accounts"
+	"github.com/prysmaticlabs/prysm/v3/cmd/validator/flags"
+	"github.com/prysmaticlabs/prysm/v3/config/features"
+	"github.com/prysmaticlabs/prysm/v3/runtime/tos"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	BeaconHostFlag = &cli.StringFlag{
+		Name:  "beacon-node-host",
+		Usage: "host:port for beacon node to query",
+		Value: "127.0.0.1:3500",
+	}
+	PathFlag = &cli.StringFlag{
+		Name:    "path",
+		Aliases: []string{"p"},
+		Usage:   "path to the signed withdrawal messages JSON",
+	}
+	ConfirmFlag = &cli.BoolFlag{
+		Name:    "confirm",
+		Aliases: []string{"c"},
+		Usage: "WARNING:User confirms and accepts responsibility of all input data provided and actions for setting setting their withdrawal address for their validator key. " +
+			"This action is not reversible and withdrawal addresses can not be changed once set. ",
+	}
+)
+
+var Commands = []*cli.Command{
+	{
+		Name:    "validator",
+		Aliases: []string{"v"},
+		Usage:   "commands that affect the state of validators such as exiting or withdrawing",
+		Subcommands: []*cli.Command{
+			{
+				Name:    "withdraw",
+				Aliases: []string{"w"},
+				Usage:   "Assign Ethereum withdrawal addresses to validator keys. WARNING: once set values are included they can no longer be updated.",
+				Flags: []cli.Flag{
+					BeaconHostFlag,
+					PathFlag,
+					ConfirmFlag,
+					cmd.ConfigFileFlag,
+					cmd.AcceptTosFlag,
+				},
+				Before: func(cliCtx *cli.Context) error {
+					if err := cmd.LoadFlagsFromConfig(cliCtx, cliCtx.Command.Flags); err != nil {
+						return err
+					}
+					au := aurora.NewAurora(true)
+					if !cliCtx.Bool(cmd.AcceptTosFlag.Name) || !cliCtx.Bool(ConfirmFlag.Name) {
+						fmt.Println(au.Red("===============IMPORTANT==============="))
+						fmt.Println(au.Red("Please read the following carefully"))
+						fmt.Print("This action will allow the partial withdraw of amounts over the 32 staked eth in your active validator balance. \n" +
+							"You will also be entitled to the full withdrawal of the entire validator balance if your validator has exited. \n" +
+							"The partial and full withdrawal . \n" +
+							"Please navigate to our website and make sure you understand the full implications of setting your withdrawal address. \n")
```

Comment:
Do we want to add a link to the website?

Reply:
we don't have a page yet and it might age quickly. I can add the prysm docs website though.